Add map example to splat.mdx (#32586)
hashicorp/terraform: website/docs/language/expressions/splat.mdx

[`for` expressions](/language/expressions/for).

Resources that use the `for_each` argument will appear in expressions as a map
of objects, so you can't directly use splat expressions with those resources.
For more information, see
[Referring to Resource Instances](/language/meta-arguments/for_each#referring-to-instances).

> It is however possible to use the [`values`](/language/functions/values) function to get the

Reviewer: ```suggestion
You can, however, use the [`values`](/language/functions/values) function to get the
```
